### DISCUSSION TITLE
How about we remove the default about section from the native menu.

### DISCUSSION THREAD

**Discussion by jhimes144** [AvaloniaUI/Avalonia#9967] [Q&A] [answered]
I propose we remove the hard-coded default about native menu item that gets created. The only way to override the default menu item is if the developer specifies a default menu in App.xaml. If defined anywhere else, their menu is tacked on with the default about menu item staying. Its very opaque how the default about menu item gets created and took me a while to figure out how to get rid of it, as I was creating native menus per window.

Instead, I propose for all templates to have a NativeMenu declared in App.xaml with the about section.

**Comment by maxkatz6** [comment, marked as answer]
I believe you want to disable default items:
```
.UsePlatformDetect()
.With(new MacOSPlatformOptions
{
   DisableDefaultApplicationMenuItems = true
})
```

**Reply by jhimes144** [reply]
Perfect thanks!
